board/freescale/imx8mm_evk/spl.c

```diff
@@ -326,24 +326,3 @@ void board_init_f(ulong dummy)
 
 	board_init_r(NULL, 0);
 }
-
```

Comment:
Wow! I've never seen this part of that commit. Pretty strange :(
Thanks for rechecking!